docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,33 @@ Release Notes for BIG-IP Controller for Kubernetes
 Next Release
 ------------
 
+1.14.0
+------------
+Added Functionality
+`````````````````````
+* Added TLSv1.3 support in CIS, by default TLSv1.3 is disabled in CIS, this can be configured using below command line arguments:
+    -  `--tls-version` to enable specific TLS version(v1.2 or v1.3) on BIG-IP
```

Comment:
Please change it 
- `--tls-version` to enable specific TLS version 1.3 on BIG-IP. Default TLS version  is 1.2.